zcash_primitives/src/sapling/note_encryption.rs

```diff
@@ -132,12 +140,14 @@ impl<P: consensus::Parameters> Domain for SaplingDomain<P> {
     // points must not be small-order, and all points with non-canonical serialization
     // are small-order.
     type EphemeralPublicKey = jubjub::ExtendedPoint;
+    type PreparedEphemeralPublicKey = PreparedBase;
```

Comment:
From @str4d: Add newtypes wrapping these so that the internal representation of these types can change without breaking the public API.

Reply:
Done.